packages/block-library/src/query-loop/index.php

```diff
@@ -57,6 +57,10 @@ function render_block_core_query_loop( $attributes, $content, $block ) {
 		)->render( array( 'dynamic' => false ) );
 		$content      .= "<li>{$block_content}</li>";
 	}
+
+	if ( ! $content ) {
```

Comment:
You could move `$wrapper_attributes` after this `if` statement now.

Reply:
You're so right :). This proves I should spent a couple of minutes more in issues I think it's just seconds. I pushed a change that checks the `posts` above. Thanks Grzegorz!